### ISSUE TITLE
Adding feeds does nothing at all

### ISSUE THREAD

**Issue by untitaker** [silverbucket/dogfeed#55]
Basically if I hit "add feed" on /feeds/add, the button greys out. Sometime "fetching feeds from ..." also gets displayed, but apart from that, nothing happens. No error or any other new log entry in the console, nothing on the network.

Firefox 40.0.3, but behavior in Chromium seems to be the same.

**Comment by silverbucket** [comment]
Yeah, this is a known issue. I hope to be able to update the code after the new version of sockethub is out.
